packages/angular_devkit/core/src/virtual-fs/host/interface.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { Observable } from 'rxjs/Observable';
+import { Path, PathFragment } from '../path';
+
+
+export declare type FileBuffer = ArrayBuffer;
+
+export interface HostWatchOptions {
+  readonly persistent?: boolean;
+  readonly recursive?: boolean;
```

Comment:
We'll probably need polling as well in the future (Docker etc).

Reply:
Definitely, but let's get this in so we can use it.